packages/template-retail-react-app/app/ssr.js

```diff
@@ -21,7 +21,7 @@ const options = {
 
     // The path to the favicon. This must also appear in
     // the mobify.ssrShared section of package.json.
-    faviconPath: path.resolve(process.cwd(), 'build/static/ico/favicon.ico'),
+    faviconPath: path.resolve(process.cwd(), (isRemote() ? 'build/static/ico/favicon.ico' : 'app/static/ico/favicon.ico')),
```

Comment:
this works, but isn't ideal! this increases the complexity in the user land, and we could move it into the SDK.